client/src/screens/groups/NewGroup.js

```diff
@@ -188,9 +190,12 @@ applyTagSearchFilter = () => {
           visible={this.state.tagsModal}
           closeModal={this.handleTagBack}
           onSearch={text => this.searchTagOnPress(text)}
+          onSelect={option => this.handleTagChange(option)}
```

Comment:
`onSearch`, `onSelect` and `backModal` can all drop the closure like this:
```
onSelect={this.handleTagChange}
```

Reply:
well thats cool. TIL